launch/axis.launch

```diff
@@ -15,7 +15,7 @@
     <node pkg="image_transport" type="republish" name="republish"
           args="compressed" if="$(arg enable_theora)">
       <remap from="in" to="image_raw" />
-      <remap from="out" to="image_raw" />
+      <remap from="out" to="image_raw_out" />
```

Comment:
I've tested it.  Previously, if I did a rostopic info
/axis/image_raw/compressed, I saw axis/republish as both a subscriber and a
publisher.  Now it doesn't do that.  I can view both the compressed and
theora topics without issue.

I should note that this only crops up if you enable theora, which makes
sense.

On Fri, Feb 6, 2015 at 10:53 AM, Mike Purvis <notifications@github.com>
wrote:

> In launch/axis.launch
> <https://github.com/clearpathrobotics/axis_camera/pull/31#discussion_r24249096>
> :
>
> > @@ -15,7 +15,7 @@
> >      <node pkg="image_transport" type="republish" name="republish"
> >            args="compressed" if="$(arg enable_theora)">
> >        <remap from="in" to="image_raw" />
> > -      <remap from="out" to="image_raw" />
> > +      <remap from="out" to="image_raw_out" />
>
> Are you sure this does the right thing?
>
> I thought the issue was the compressed topic overlapping?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/clearpathrobotics/axis_camera/pull/31/files#r24249096>
> .
>



-- 
Jeff Schmidt
System Integration Technologist
Clearpath Robotics
Phone: 519-513-2416